### PR TITLE
Add notes for 03-11-2019 TSC meeting

### PR DESCRIPTION
**Pull request by carl-foundry**

### CODE REVIEW

File: docs/tsc/meetings/2019-03-11.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+Mar 11, 2019
+
+Host: Michael Dolan
+
+Rotating Secretary: Carl Rand
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [ ] Sean Cooper (_TSC_) - DNEG
```
Comment (michdolan) [inline docs/tsc/meetings/2019-03-11.md]:
```suggestion
  * [X] Sean Cooper (_TSC_) - DNEG
```

Sean was present.

File: docs/tsc/meetings/2019-03-11.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  * [X] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+
+# **OCIO TSC Meeting Notes**
+
+* Prerequisites to moving the repository to ASWF
+    - Resolve "ext/" dependencies prior to the move.
+    - What are we going to do with the "ext/" directory?
+    - Do we maintained or replace?
```
Comment (michdolan) [inline docs/tsc/meetings/2019-03-11.md]:
```suggestion
    - Do we maintain or replace?
```

File: docs/tsc/meetings/2019-03-11.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    - John: Resolving issues with the initial setup. Team is currently backlogged on other projects but hoping to get something done in the next couple of days.
+    - DCO when we are ready to DCO sign off with commits.
+    - Every commit will have DCO sign off.
+    - John: Some info can be pulled or referenced from the TAC guidelines.
+    - Larry: Who needs to sign off the work?
+        - John: We can refer to ​https://developercertificate.org/
+    - Michael: This isn’t required until we move to the new account. Not to worry about it for now.
+    - Want to have the Linux foundation CLA system up and running first, pre-sign all the CLAs, and do one commit to flick the switch.
+    - Larry: Have agreed on the CLA but not on all the surrounding areas e.g. signature blocks etc.
+
```
Comment (michdolan) [inline docs/tsc/meetings/2019-03-11.md]: Could add:

`- **TODO**: Document DCO sign off practices in CONTRIBUTING.md`

File: docs/tsc/meetings/2019-03-11.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+* Googlegroup archiving & updated gh-pages deployment
+    - Announced we will pull down the group in a week.
+    - A script will be run to get old contend and re-import it into the new group.
```
Comment (michdolan) [inline docs/tsc/meetings/2019-03-11.md]:
```suggestion
    - A script will be run to get old content and re-import it into the new lists.
```

File: docs/tsc/meetings/2019-03-11.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+* Next 1.1.x release
+    - There are a small number of commits that will need to be signed off before the release is made.
+    - Agreed to call it the 1.1.1 release.
+    - It will be the first release of the year.
+    - **TODO**: Michael to finalize things to push it forward.
+    - **TODO**: Will need to update the change log for the new version.
+
+* Items for next TSC meeting agenda:
+    - xxx
```
Comment (michdolan) [inline docs/tsc/meetings/2019-03-11.md]: If there's nothing here, we can delete this section.

File: docs/tsc/meetings/2019-03-11.md (hunk 1 above)
Comment (carl-foundry) [reply docs/tsc/meetings/2019-03-11.md]: Correction made in commit 519694f4b44c3d9a5e57f5084f221329b4734236

File: docs/tsc/meetings/2019-03-11.md (hunk 2 above)
Comment (carl-foundry) [reply docs/tsc/meetings/2019-03-11.md]: Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236

File: docs/tsc/meetings/2019-03-11.md (hunk 3 above)
Comment (carl-foundry) [reply docs/tsc/meetings/2019-03-11.md]: Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236

File: docs/tsc/meetings/2019-03-11.md (hunk 4 above)
Comment (carl-foundry) [reply docs/tsc/meetings/2019-03-11.md]: Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236

File: docs/tsc/meetings/2019-03-11.md (hunk 5 above)
Comment (carl-foundry) [reply docs/tsc/meetings/2019-03-11.md]: Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236

**Pull request merged**